.cargo/config.toml

```diff
@@ -1,5 +1,5 @@
-[build]
-target = "wasm32-unknown-unknown"
```

Comment:
We have to drop this, otherwise the xtask gets built as `wasm32-unknown-unknown`, and therefore won't run.

It was only really here as a convenience for people opening the project in their IDE anyway. There are better ways to deal with that.